nifi-extension-bundles/nifi-amqp-bundle/nifi-amqp-processors/src/main/java/org/apache/nifi/amqp/processors/AbstractAMQPProcessor.java

```diff
@@ -139,23 +139,16 @@ abstract class AbstractAMQPProcessor<T extends AMQPWorker> extends AbstractProce
             .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
             .build();
 
-    private static final List<PropertyDescriptor> propertyDescriptors;
-
-    static {
-        propertyDescriptors = List.of(
-                BROKERS,
-                HOST, PORT,
-                V_HOST,
-                USER,
-                PASSWORD,
-                AMQP_VERSION,
-                SSL_CONTEXT_SERVICE,
-                USE_CERT_AUTHENTICATION);
-    }
-
-    protected static List<PropertyDescriptor> getCommonPropertyDescriptors() {
-        return propertyDescriptors;
-    }
+    protected static final List<PropertyDescriptor> PARENT_PROPERTIES = List.of(
+            BROKERS,
+            HOST, PORT,
+            V_HOST,
+            USER,
+            PASSWORD,
+            AMQP_VERSION,
+            SSL_CONTEXT_SERVICE,
+            USE_CERT_AUTHENTICATION
+    );
```

Comment:
Here the parent class 'getCommonPropertyDescriptors' method is removed and instead a subclass is supposed to know to use the static list of descriptors as they build their own lists.  I think this weakens what the author intended in building that class.

Also there is at least one example much later on in this PR where you kept the parent method.  I think keeping the parent method better conveys intent to any subclassers.

Reply:
I will revert this back.